app/src/main/java/org/xbmc/kore/ui/generic/NavigationDrawerFragment.java

```diff
@@ -81,9 +82,11 @@
             ACTIVITY_TVSHOWS = 3,
             ACTIVITY_MUSIC = 4,
             ACTIVITY_PVR = 5,
-            ACTIVITY_FILES = 6,
-            ACTIVITY_ADDONS = 7,
-            ACTIVITY_SETTINGS = 8;
+            ACTIVITY_FAVOURITES = 6,
+            ACTIVITY_FILES = 7,
+            ACTIVITY_ADDONS = 8,
+            ACTIVITY_SETTINGS = 9;
+
```

Comment:
I tried to use the first suggestion, because I'm afraid that if I went with the second one, I may introduce more bugs.


So to be in the safe side I restored the previous order and assigned 9 to ACTIVITY_FAVOURITES, then added 9 to the `entry_values_nav_drawer_items` in arrays.xml. 

But I've noticed that the number of items in the `entries_nav_drawer_items` doesn't match the number of items in `entry_values_nav_drawer_items`. 

Do you know what is the reason behind that? Because I tried adding "Favourites" under `entries_nav_drawer_items` but noticed the mismatch between the index of that entry and the corresponding index at `entry_values_nav_drawer_items`.